### ISSUE TITLE
create simple models for posts and comments

### ISSUE THREAD

**Issue by pdpinch** [mitodl/open-discussions#730]
Due to the 1000 item limit in reddit APIs, we need a way to keep track of posts and comments that we want to index for searching. The idea is that we would make note of the post/comment id whenever one is created and instead of relying on reddit to return a list of all posts in a channel/comments on a post, we could walk this table. 

This may be the starting point of a larger-scale replacement of reddit for keeping track of data for search. 

Before we start on this, we need a Google doc describing the schema and scope. 

### acceptance criteria
- [ ] [Google doc describing the plan](https://docs.google.com/document/d/1V311dVEDBcX3zNjmDVe-mcMxoUQL3ChCVGkp7CxsGzA/edit#heading=h.db6yb9iglzuh)
- [ ] create a new post/comment when user posts/comments
- [ ] walk table when creating index instead of traversing reddit API

### out of scope
- [ ] storing other fields that are indexed
- [ ] updating post/comment when user updates

**Comment by pdpinch** [comment]
Please read and comment on https://docs.google.com/document/d/1V311dVEDBcX3zNjmDVe-mcMxoUQL3ChCVGkp7CxsGzA/edit#heading=h.db6yb9iglzuh before starting work on this.

**Comment by pdpinch** [comment]
We agreed that this issue would be limited to storing ids for posts, comments and channels. 

We are considering follow-up work to store complete post and comment data in SQL.
